ai/STA/Tactic/position_for_pass.py

```diff
@@ -24,7 +25,7 @@
 class PositionForPass(Tactic):
 
     def __init__(self, game_state: GameState, player: Player, target: Pose=Pose(), args: List[str]=None,
-                 auto_position=False):
+                 auto_position=False, robots_in_formation: List[Player] = None):
```

Comment:
should be: `... args: Optional[List[str]]=None, auto_position: bool=False, robots_in_formation: Optional[List[Player]]=None`